app/views/state_file/questions/shared/_id_retirement_income_deductions_review_header.html.erb

```diff
@@ -1,13 +1,84 @@
-<% if current_intake.calculator.line_or_zero(:ID39R_B_LINE_8e).positive? %>
+<% if current_intake.show_disability_question? %>
+  <section id="disability-info" class="white-group">
+    <p class="text--bold spacing-below-5"><%= t("state_file.questions.shared.id_disability_review_header.meets_qualifications") %></p>
+    <p class="spacing-below-15"><%= (current_intake.primary_disabled_yes? || current_intake.spouse_disabled_yes?) ? t("general.affirmative") : t("general.negative") %></p>
+    <p class="text--small spacing-below-15"><%= t("state_file.questions.shared.id_disability_review_header.editing_may_affect_deductions") %></p>
+    <div class="spacing-below-5">
+      <%= link_to t("general.review_and_edit"), StateFile::Questions::IdDisabilityController.to_path_helper(
+        return_to_review_before: StateFile::Questions::IdDisabilityController.name.demodulize.underscore,
+        return_to_review_after: "retirement_income_deduction"), class: "button--small" %>
```

Comment:
the way I understand this after looking at it -- `return_to_review_before` and `return_to_review_after` are kind of bookend markers: if you reach at this controller when you click to go prev/back, return to review & if you reach end of "retirement_income_deduction" when you click to proceed/forward, return to review.